conda-recipes/llvmlite/meta.yaml

```diff
@@ -37,7 +37,7 @@ requirements:
     # llvmdev is built with libz compression support
     - zlib           # [unix and not (armv6l or armv7l)]
   run:
-    - python >=3.7,<3.10
+    - python >=3.8,<3.10
```

Comment:
I noticed this as well and was wondering why this CI config was not doing it for Python 3.10 even though support was added back in https://github.com/numba/llvmlite/pull/769.

Reply:
My guess is that the conda build is ignoring / overwriting this constraint because it needs to generate one for the Python version built for anyway. For example, the `run` constraint from the package it builds for Python 3.10 ends up looking like `python >=3.10,<3.11.0a0`.

Reply:
@gmarkall thank you for pointing this out.

It was introduced in: https://github.com/numba/llvmlite/pull/747/files

And not removed with: 

https://github.com/numba/llvmlite/pull/769/files

As stated, I can confirm that:

```
tar xOf llvmlite-0.40.0dev0llvm14-py310_48.tar.bz2 info/recipe/meta.yaml
```

Will show:

```
...
  run:
    - libcxx >=4.0.1
    - python >=3.10,<3.11.0a0
    - zlib >=1.2.13,<1.3.0a0
...
```

So probably, this ends up not having any effect. So, in this case I would tend to lean towards removing the limit entirely, either in this PR or in a new one.